src/pages/Beneficiary.tsx

```diff
@@ -39,68 +67,68 @@ const Beneficiary = () => {
   };
 
   return (
-    <div>
-      <HeaderAndFooterWrapper>
-        <Grid container>
-          <Grid item xs={10}>
-            <Typography> Beneficiary </Typography>
+    <MainBgDiv>
+      <MainInnerDiv>
+        <HeaderAndFooterWrapper>
+          <Grid container>
+            <Grid item xs={10}>
+              <Typography> Beneficiary </Typography>
+            </Grid>
+            <Grid item xs={2}>
+              <FullScreenDialogCustom
+                handleClick={() => {}}
+                open={open}
+                setOpen={handleOpen}
+                title="Register New Beneficiary"
+                mainLayout={<RegisterBeneficiary setOpen={setOpen} />}
+              >
+                <Button style={{ backgroundColor: "green", color: "white" }}>
+                  {" "}
+                  Register{" "}
+                </Button>
+              </FullScreenDialogCustom>
+            </Grid>
           </Grid>
-          <Grid item xs={2}>
-            <FullScreenDialogCustom
-              handleClick={() => {}}
-              open={open}
-              setOpen={handleOpen}
-              title="Register New Beneficiary"
-              mainLayout={<RegisterBeneficiary setOpen={setOpen} />}
-            >
-                <Button style={{backgroundColor:"green", color:'white'}}> Register </Button>
-            </FullScreenDialogCustom>
-          </Grid>
-        </Grid>
 
-   
-        <Grid container spacing={2} justifyContent="center" style={{marginTop:'20px'}}>
-          {beneficiaryList.length > 0 &&
-            beneficiaryList.map((val, key) => {
-              return (
-                <Grid
-                  item
-                  xs={12}
-                  sm={6}
-                  md={4}
-                  lg={3}
-                  key={key}
-                  onClick={() => handleSelect(val)}
-                >
-                  <div
-                    style={{
-                      boxShadow: "rgb(0 1 10 / 5%) 0px 5px 10px",
-                      background: "#FFF",
-                      borderRadius: "1rem",
-                      border: "2px solid green",
-                      marginBottom: "40px",
-                      height: "100%",
-                      margin: "10px",
-                      // position: "relative",
-                      // display: "flex",
-                    }}
+          <Grid
+            container
+            spacing={2}
+            justifyContent="center"
+            style={{ marginTop: "20px" }}
+          >
+            {beneficiaryList.length > 0 &&
```

Comment:
Can be simplified to:

```suggestion
            {beneficiaryList.length &&
```